src/status_im/protocol/web3/inbox.cljs

```diff
@@ -44,10 +44,10 @@
                               (let [args {:jsonrpc "2.0"
                                           :id      2
                                           :method  "shh_requestMessages"
-                                          :params  [{:peer      enode
-                                                     :topic     topic
-                                                     :symKeyID  sym-key-id
-                                                     :from      0}]}
+                                          :params  [{:mailServerPeer enode
+                                                     :topic          topic
+                                                     :symKeyID       sym-key-id
+                                                     :from           0}]}
```

Comment:
`from: 0` can be skipped as missing params get zero-values automatically. 

Also, I guess this will be refactored, right? Not sure how the logic in status-react looks like but I imagine that messages will be requested each time the app gets online. If a user has a bad connection, it will request messages from the last 7 days over and over again.

Reply:
@adambabik thanks, i'll remove `from` parameter then.
Now the logic is that app requests historic messages on "account login" and "switch node" events.  Not sure app handles bad connection case right now, seems like it does not. I will check this case as well.